### PR TITLE
Update to instructor notes

### LINKED ISSUE #345: Update instructor notes
Update instructor notes - they are a bit outdated.

### PR DESCRIPTION
**Pull request by anenadic**
Fixes #345

### CODE REVIEW

File: _extras/guide.md [hunk 1]
```diff
@@ -61,15 +61,15 @@ The course can be delivered online or in-person.
...
+While this course can be live-coded by an instructor as well,
 we felt that intermediate-level learners are capable of
 going through the material on their own at a reasonable speed
 and would not require to code-along to the same extent as novice learners.
 In later stages, exercises require participants to develop code more individually
 so they can review and comment on each other's code,
 so the codes need to be sufficiently different for these exercises to be effective.
-Having an instructor live-code would make everyone have exactly the same code on their machines
-and would not have the same effect.
+For instrctor-led mode of delivery, you can have an instructor live-code these group exercises
```
Comment (steve-crouch) [inline _extras/guide.md:71]:
```suggestion
For instructor-led mode of delivery, you can have an instructor live-code these group exercises 
```

File: _extras/guide.md [hunk 2]
```diff
@@ -61,15 +61,15 @@ The course can be delivered online or in-person.
...
 we felt that intermediate-level learners are capable of
 going through the material on their own at a reasonable speed
 and would not require to code-along to the same extent as novice learners.
 In later stages, exercises require participants to develop code more individually
 so they can review and comment on each other's code,
 so the codes need to be sufficiently different for these exercises to be effective.
-Having an instructor live-code would make everyone have exactly the same code on their machines
-and would not have the same effect.
+For instrctor-led mode of delivery, you can have an instructor live-code these group exercises 
+after learner have been given a chance to work on them as a team.
```
Comment (steve-crouch) [inline _extras/guide.md:72]:
```suggestion
after learners have been given a chance to work on them as a team.
```

File: _extras/guide.md [hunk 3]
```diff
@@ -79,17 +79,19 @@ to explain what the students will be learning and doing on that particular day.
...
+If participants have not finished all exercises for a section (in "self-learning with helpers" mode),
 they are asked to finish them off before the next section starts
 to make sure everyone is in sync as much as possible and are working on similar things
 (though students will inevitably cover the material at different speeds).
 This synchronisation becomes particularly important for later workshop stages
 when students start with group exercises.
 
 ### Helpers Roles and Responsibilities
-At the workshop, everyone in the training team is a helper.
+At the workshop, "self-learning with helpers" mode everyone in the training team is a helper and
```
Comment (steve-crouch) [inline _extras/guide.md:90]:
```suggestion
At the workshop, when using the "self-learning with helpers" delivery mode, everyone in the training team is a helper and 
```

File: _extras/guide.md [hunk 4]
```diff
@@ -79,17 +79,19 @@ to explain what the students will be learning and doing on that particular day.
...
 they are asked to finish them off before the next section starts
 to make sure everyone is in sync as much as possible and are working on similar things
 (though students will inevitably cover the material at different speeds).
 This synchronisation becomes particularly important for later workshop stages
 when students start with group exercises.
 
 ### Helpers Roles and Responsibilities
-At the workshop, everyone in the training team is a helper.
+At the workshop, "self-learning with helpers" mode everyone in the training team is a helper and 
+there are no insturctors per se.
```
Comment (steve-crouch) [inline _extras/guide.md:91]:
```suggestion
there are no instructors per se.
```

File: _extras/guide.md [hunk 5]
```diff
@@ -79,17 +79,19 @@ to explain what the students will be learning and doing on that particular day.
...
 You may have more experienced helpers delivering introductions to the workshop and sections.
-Contact the course authors for intro slides you can reuse.
+Contact the course authors for the section intro slides you can reuse and you can also find slides for each
```
Comment (steve-crouch) [inline _extras/guide.md:93]:
```suggestion
Contact the course authors for section intro slides you can reuse, and you can also find slides for each 
```

File: _extras/guide.md [hunk 6]
```diff
@@ -79,17 +79,19 @@ to explain what the students will be learning and doing on that particular day.
...
 when students start with group exercises.
 
 ### Helpers Roles and Responsibilities
-At the workshop, everyone in the training team is a helper.
+At the workshop, "self-learning with helpers" mode everyone in the training team is a helper and 
+there are no insturctors per se.
 You may have more experienced helpers delivering introductions to the workshop and sections.
-Contact the course authors for intro slides you can reuse.
+Contact the course authors for the section intro slides you can reuse and you can also find slides for each 
+section in the course repository (for intructor-led delivery mode).
```
Comment (steve-crouch) [inline _extras/guide.md:94]:
```suggestion
section in the course repository (for instructor-led delivery mode).
```

**Review by steve-crouch — CHANGES_REQUESTED** [review]
Looks good, just some minor typos really.

**Review by anenadic — COMMENTED** [review]
Fab - many thanks for the review and all the fixes @steve-crouch

**Review by steve-crouch — APPROVED** [review]

**Pull request merged**